adminSiteClient/GrapherConfigGridEditor.tsx

```diff
@@ -0,0 +1,1267 @@
+            this.desiredPagingOffset + PAGEING_SIZE < (this.numTotalRows ?? 0)
+        ) {
+            this.desiredPagingOffset = this.desiredPagingOffset + PAGEING_SIZE
+        }
+    }
+
+    renderPagination(): JSX.Element {
+        const { numTotalRows, currentPagingOffset } = this
+        const currentStartLabel = currentPagingOffset + 1
+        const currentEndLabel = Math.min(
+            currentPagingOffset + PAGEING_SIZE,
+            numTotalRows ?? 0
+        )
+
+        return (
+            <nav aria-label="Variable annotation pagination controls">
+                <ul className="pagination">
+                    <li className="page-item">
+                        <a
+                            className="page-link"
+                            href="#"
+                            aria-label="Previous"
+                            onClick={this.pageToPreviousPage}
+                        >
+                            <span aria-hidden="true">&laquo;</span>
+                        </a>
+                    </li>
+                    <li className="page-item">
+                        <a className="page-link">
+                            {currentStartLabel} to {currentEndLabel} of{" "}
+                            {numTotalRows}
+                        </a>
+                    </li>
+                    <li className="page-item">
+                        <a
+                            className="page-link"
+                            href="#"
+                            aria-label="Next"
+                            onClick={this.pageToNextPage}
+                        >
+                            <span aria-hidden="true">&raquo;</span>
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+        )
+    }
+
+    renderFilterTab(): JSX.Element {
+        const { FilterPanelConfig, filterState } = this
+        return (
+            <section>
+                <div className="container">
+                    <h3>Variable filters</h3>
+                    {this.renderPagination()}
+                    {this.source ===
+                    GrapherConfigGridEditorSource.SourceVariableAnnotation ? (
+                        <React.Fragment>
+                            <BindString
+                                field="namespaceNameFilter"
+                                store={this}
+                                label="Namespace name"
+                            />
+                            <BindString
+                                field="datasetNameFilter"
+                                store={this}
+                                label="Dataset name"
+                            />
+                            <BindString
+                                field="variableNameFilter"
+                                store={this}
+                                label="Variable name"
+                            />
```

Comment:
We can probably get rid of these, now that all of this can also be achieved using the query builder?
That simplifies the handling of these above.